edb/edgeql/compiler/stmt.py

```diff
@@ -132,6 +132,7 @@ def compile_ForQuery(
 
             iterator_view = stmtctx.declare_view(
                 iterator, qlstmt.iterator_alias,
+                temporary=False,
```

Comment:
`scopectx` should probably be `iterator_ctx` here (assuming `else: iterator_ctx = scopectx` is added above)

Reply:
OK, I'll try that. Basically every single thing about this PR has changed since that version, though :P